docs/tools/idx/overview.md

```diff
@@ -1,5 +1,10 @@
 # IDX
 
+!!! warning "Deprecation notice"
+
+    :octicons-stop-16: The reference implementations for the Identity Index (IDX) specification are now provided by the [Glaze](../glaze/overview.md) and [Self.ID](../self-id/overview.md) projects.
```

Comment:
Should this refer to `did-datastore` specifically?

Reply:
Specifically yes but I think it's more useful to go through the overview first, no?

Reply:
I mean just this sentence could say did-datastore, which is provided by glaze/self.id

Reply:
OK I see, thanks! I updated the message, is it better now?

Reply:
yes, thanks!